content/guides/accessibility-for-teams/content-design.md

```diff
@@ -9,124 +9,109 @@ primary_image: guide-dap
 layout: single
 ---
 
-# Getting started
-
-## Accessibility for content designers
-
 Accessible writing ensures your content is easier for everyone to read. As we build government services, we want to ensure they are accessible and welcoming to everyone who needs to use them.
 
-### How to use this guide
+## How to use this guide
 
 - We recommend conducting accessibility testing throughout the design and development processes.
 - If you have project-specific questions, ask your agency’s accessibility team.
 
-
-# Plain language
-
 ## Plain language
 
 {{< ring title="Why it's important" >}}
-
 Can you quickly understand the main points of the content?
 
 - Karin is not a native English speaker and she sometimes has trouble decoding legal or bureaucratic language.
 - John has a developmental disability and has difficulty interpreting content written above a sixth-grade reading level.
 - Kai has low tech literacy and often doesn’t understand highly technical language.
-
 {{< /ring >}}
 
 ### Steps to take
+
 - Refer to the [plain language section](https://pages.18f.gov/content-guide/plain-language/) of 18F’s Content Guide for general guidance, lists of words to avoid, and links to plain-language resources.
 - As you’re writing, consider the tech literacy level of your target audience. Define technical terms that may be unfamiliar, and use a product or service’s full name before using its acronym or abbreviation. You may also consider [adding a glossary](https://github.com/18F/glossary) if your content contains many potentially unfamiliar terms.
 Include in-line definitions for scientific, legal, or technical terms that you must use.
 - Avoid using idiomatic language.
 - Test the readability of your content using [Hemingway App](http://www.hemingwayapp.com/), [Readable.io](https://readable.io/), or a similar tool.
 
-#### Resources
-[plainlanguage.gov](https://plainlanguage.gov/guidelines/)
+### Resources
+
+  - [plainlanguage.gov](https://plainlanguage.gov/guidelines/)
 
-#### WCAG 2.0 references
+{{< note variant="wcag" >}}
 - [3.1 Readable (Guideline)](https://www.w3.org/WAI/WCAG20/quickref/#meaning)
 - [3.1.3 Unusual Words](https://www.w3.org/WAI/WCAG20/quickref/#meaning-idioms)
 - [3.1.4 Abbreviations](https://www.w3.org/WAI/WCAG20/quickref/#meaning-located)
 - [3.1.5 Reading Level](https://www.w3.org/WAI/WCAG20/quickref/#meaning-supplements)
+{{< /note >}}
 
 
-# Reference materials
-
 ## Reference materials
 
 {{< ring title="Why it's important" >}}
-
 Can you easily access supplementary information clarifying the content?
 
 - Gilbert reads at a twelfth-grade level but isn’t familiar with the nuances of a site’s subject matter; to fully understand the site content, he needs easy-to-access contextual information.
-
 {{< /ring >}}
```

Comment:
TODO: We should modify the `ring` shortcode to take a header. Currently it displays a `h2` and breaks the guide structure.